src/wot.iced

```diff
@@ -65,7 +65,15 @@ exports.Vouch = class Vouch extends Base
       vouch_text : schema.array(schema.string())
     })
     err = schm.check(obj)
-    cb err
+    if err?
+      cb err
+    
+    for text in obj.vouch_text
```

Comment:
instead of this probably add to the string schema so you can do `schema.string_with_max_length(700))` above and get it for free

that being said, this cb stuff is just continuation-passing so you have to return whenever you do a cb unless its at the end of the function, so you can fix this code with s/cb/return cb